src/image-sync-manager.ts

```diff
@@ -24,25 +37,29 @@ export class ImageSyncManager {
 
   private _syncEvents = () => {
     this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA, this._onTimedMetadata);
-    this._eventManager.listen(this._mainPlayer, TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
+    this._eventManager.listen(this._mainPlayer, this._mainPlayer.cuePointManager.TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
   };
 
-  private _onTimedMetadata = ({payload}: {payload: {cues: Array<{track: {label: string; language: string}; value: {data: {id: string}}}>}}) => {
-    if (payload.cues[0]?.track?.label === 'KalturaCuePoints' && payload.cues[0]?.track?.language === SLIDE_CUE_POINT_TYPE) {
+  private _onTimedMetadata = ({payload}: TimedMetadata) => {
+    if (
+      payload.cues[0]?.track?.label === this._mainPlayer.cuePointManager.CuePointsTextTrack &&
```

Comment:
the consts should be under a context (like the event types under `Event`). same for CuePointKey

Reply:
maybe it could be an object exposed by kaltrua player index.js. like `playlist` see https://github.com/kaltura/kaltura-player-js/blob/master/src/index.js#L50 and https://github.com/kaltura/kaltura-player-js/blob/master/src/common/playlist/index.js